src/tex/ms.tex

```diff
@@ -216,8 +218,11 @@ \section{Posterior}
 \end{figure}
 %
 
-To report values for parameters and some form of statistical uncertainty, two clear approaches can be taken from the posterior chain. 
-The first is to use some known statistical distribution that describes the samples well. 
+To report values for parameters and some form of statistical uncertainty, two approaches can be taken from the posterior chain. 
+The first is to use some known statistical distribution that describes the samples well.
+% ARJN I think this will rapidly become intractible. For example what would one do with e.g. 100 parameters, should one be checking them all for normality? There will inevitably ones that are skewed. Then there will be a hodgepodge of answers.
```

Comment:
Which part is intractable? Checking normality?

Reply:
- If you have hundreds of parameters it'll be a bind to go through them all.
- It's not clear to me why checking for normality is important. The only reason to check for normality is if you are predicting that a parameter would be distributed normally, and you want to check for deviation from normality.

Reply:
Checking normality is important (to me) because, the majority of users will go `chain.mean(0)` and `chain.std(0)` and quote these values, but the latter is useless if the distribution is not normal.